sickbeard/providers/tvchaosuk.py

```diff
@@ -102,7 +100,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
             for search_string in search_strings[mode]:
 
                 if mode == 'Season':
-                    search_string = re.sub(ur'(.*)Season', ur'\1Series', search_string)
+                    search_string = re.sub(ur'(.*)S0?', ur'\1Series', search_string)
```

Comment:
Would it bring any advantage? The search string ever has multiple seasons? 
With the current one it will only replace `S` to `Series` and if its a season shorter than 9 `S0` to Series becouse in the site all seasons less than 10 dont have a 0 and if its searched with it no results will be returned
Also there arent any seasons with year from my research in the site